R/calculate_within_batch_ari.R

```diff
@@ -2,7 +2,10 @@
 #'
 #'
 #' @param individual_sce_list A list of individual SCE objects
-#' @param pc_name The name that allows access to the PCs. Example: fastMNN_PCA
+#' @param individual_pc_name The name that allows access to the PCs in the
+#'  individual SCE objects. Default is "PCA".
+#' @param integrated_pc_name The name that allows access to the PCs in the
+#'  integrated SCE object.  Example: "fastMNN_PCA".
```

Comment:
I think what we actually want to do here is just have one `pc_name` argument that is only used for the integrated object. For the individual object, I think it should be required that it has a `PCA` stored in it. There shouldn't be any other options for the individual object. 
Maybe what will make this more straight forward is referring to `integrated_sce` as `merged_sce`? And I would do that across the board for other functions that use that argument name. These should be able to be used with a merged object regardless of if there is batch corrected embeddings stored or not. 
```suggestion
#' @param pc_name The name that allows access to the PCs in the
#'  integrated SCE object.  Example: "fastMNN_PCA".
```

Reply:
I made the change from `integrated_sce` to `merged_sce` in the silhouette width script here as well 👍